app/models/admin/scholarship.rb

```diff
@@ -1,5 +1,6 @@
 class Admin::Scholarship < ApplicationRecord
-  belongs_to :group, :class_name=> 'Admin::ScholarshipGroup', inverse_of: 'admin_scholarships'
+  #belongs_to :group, :class_name=> 'Admin::ScholarshipGroup', inverse_of: 'admin_scholarships'
+  belongs_to :group, :class_name=> 'Admin::ScholarshipGroup', inverse_of: 'scholarships'
```

Comment:
Why do you do it ? Does the story tell you to it ? Please discard this changes. Because it has already an issue which already is being pending for your review.